libs/ui-toolkit/src/components/explorer-link/explorer-link.tsx

```diff
@@ -0,0 +1,53 @@
+import classNames from 'classnames';
+import type { ReactNode, HTMLProps } from 'react';
+
+const EXPLORER_URL = process.env['NX_EXPLORER_URL'] as string;
+
+type ExplorerEntity = 'block' | 'party';
+
+type ExplorerLinkProps = HTMLProps<HTMLAnchorElement> & {
+  entity: ExplorerEntity;
+  id?: string;
+  children?: ReactNode;
+};
+
+const entityUrlMap: Record<ExplorerEntity, string> = {
+  block: 'blocks',
```

Comment:
chore (blocking): transaction hash will also be needed here, `/txs`

Reply:
meanwhile, added this part, thank you! :)